diff --git a/node_modules/@react-aria/listbox/dist/main.js b/node_modules/@react-aria/listbox/dist/main.js
index 91ff221..0553673 100644
--- a/node_modules/@react-aria/listbox/dist/main.js
+++ b/node_modules/@react-aria/listbox/dist/main.js
@@ -132,7 +132,7 @@ function $c164f9f79f4cef2d$export$497855f14858aa34(props, state, ref) {
         optionProps['aria-describedby'] = descriptionId;
     }
     if (isVirtualized) {
-        optionProps['aria-posinset'] = state.collection.getItem(key).index + 1;
+        optionProps['aria-posinset'] = state.collection.getItem(key)?.index + 1 || 0;
         optionProps['aria-setsize'] = $eFCLV$reactstatelycollections.getItemCount(state.collection);
     }
     let { itemProps: itemProps , isPressed: isPressed  } = $eFCLV$reactariaselection.useSelectableItem({
diff --git a/node_modules/@react-aria/listbox/dist/module.js b/node_modules/@react-aria/listbox/dist/module.js
index 83c3ad5..d60ea68 100644
--- a/node_modules/@react-aria/listbox/dist/module.js
+++ b/node_modules/@react-aria/listbox/dist/module.js
@@ -116,7 +116,7 @@ function $293f70390ea03370$export$497855f14858aa34(props, state, ref) {
         optionProps['aria-describedby'] = descriptionId;
     }
     if (isVirtualized) {
-        optionProps['aria-posinset'] = state.collection.getItem(key).index + 1;
+        optionProps['aria-posinset'] = state.collection.getItem(key)?.index + 1 || 0;
         optionProps['aria-setsize'] = $frWMC$getItemCount(state.collection);
     }
     let { itemProps: itemProps , isPressed: isPressed  } = $frWMC$useSelectableItem({
diff --git a/node_modules/@react-aria/listbox/src/useOption.ts b/node_modules/@react-aria/listbox/src/useOption.ts
index 1d92787..a054194 100644
--- a/node_modules/@react-aria/listbox/src/useOption.ts
+++ b/node_modules/@react-aria/listbox/src/useOption.ts
@@ -122,7 +122,7 @@ export function useOption<T>(props: AriaOptionProps, state: ListState<T>, ref: R
   }
 
   if (isVirtualized) {
-    optionProps['aria-posinset'] = state.collection.getItem(key).index + 1;
+    optionProps['aria-posinset'] = state.collection.getItem(key)?.index + 1 || 0;
     optionProps['aria-setsize'] = getItemCount(state.collection);
   }
 
