diff --git a/node_modules/final-form/dist/final-form.cjs.js b/node_modules/final-form/dist/final-form.cjs.js
index 32b3ac9..7f46d5e 100644
--- a/node_modules/final-form/dist/final-form.cjs.js
+++ b/node_modules/final-form/dist/final-form.cjs.js
@@ -801,6 +801,7 @@ function createForm(config) {
 
     var notifyField = function notifyField(name) {
       var field = safeFields[name];
+      if (field == null) return
       var fieldState = publishFieldState(formState, field);
       var lastFieldState = field.lastFieldState;
       field.lastFieldState = fieldState;
diff --git a/node_modules/final-form/dist/final-form.es.js b/node_modules/final-form/dist/final-form.es.js
index 42a41bd..f6d8aa7 100644
--- a/node_modules/final-form/dist/final-form.es.js
+++ b/node_modules/final-form/dist/final-form.es.js
@@ -792,6 +792,7 @@ function createForm(config) {
 
     var notifyField = function notifyField(name) {
       var field = safeFields[name];
+      if (field == null) return
       var fieldState = publishFieldState(formState, field);
       var lastFieldState = field.lastFieldState;
       field.lastFieldState = fieldState;
